src/www/components/landingpage/LandingPage.scss

```diff
@@ -0,0 +1,612 @@
+.container {
+  background-image: linear-gradient(  rgb(129, 129, 129) ,white);
+  height: 325vh; //parallax(110vh) + 2 pages(200vh) + 15vh more empty space
+  max-width: 100vw;
```

Comment:
Does this fix something? I didn't notice any differences when removing it locally.